fitbenchmarking/fitbenchmarking
# fix options_template and add tests for usage examples

#### Description of Work

Fixes #773, fixes #774


#### Testing Instructions

1. Check that all usage examples now run without errors
2. Check that test for usage examples passes
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
